infrastructure/main.tf

```diff
@@ -53,12 +53,14 @@ locals {
 }
 
 module "db" {
-  source              = "git@github.com:hmcts/moj-module-postgres.git?ref=master"
+  source              = "git@github.com:hmcts/moj-module-postgres?ref=cnp-449-tactical"
```

Comment:
can this be merged to master?

Reply:
I suppose this has to be a controlled change. For now, we're supposed to use this one. My guess is that this branch will be merged into master once all DBs have been migrated.